README.md

```diff
@@ -117,9 +117,9 @@ in the `docker-compose.yml` file.
 The CMS key needs to be added to the admin user created by Directus.
 Otherwise, the API cannot authenticate even though its CMS key matches.
 
-- Visit http://localhost:3001
+- Visit http://localhost:8085
```

Comment:
Thanks for this! Although the port is [`8055`](https://github.com/deptagency/algomart/blob/main/apps/cms/.env.example#L4):

```suggestion
- Visit http://localhost:8055
```